doc/index.rst

```diff
@@ -66,6 +66,7 @@ Table of contents
 
    user_guide/index
    file_specification/index
+   user_guide/supported_formats
```

Comment:
Does not work currently, and is already a subcategory of user_guide. Instead add a grid to the main page with a prominent link to the table of supported formats (Maybe even before installation and any other category).

Reply:
How about a grid with six items:

* Supported Formats
* Installation
* Interoperability

and in a second row:

* File specifications
* API
* Contributing

That could replace the whole TOC and come before the splitting note and the citing section!